src/components/UserProfile/UpdateProfile/_updateProfile.scss

```diff
@@ -0,0 +1,88 @@
+.modal {
+  display: block;
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgb(0, 0, 0);
+  background-color: rgba(0, 0, 0, 0.4);
+  animation-name: animatetop;
+  animation-duration: 0.4s;
+}
+
+.modal-content {
+  background-color: #fefefe;
+  margin: 10% auto;
+  padding: 5rem;
+  border: 1px solid #888;
+  width: 50%;
+  position: fixed;
+  top:-15%;
+  left:25%
+}
+
+.close {
+  color: #aaa;
+  float: right;
+  font-size: 28px;
+  font-weight: bold;
```

Comment:
for consistency, we use rem as our font unit. in this project 28px is the same as 2.8rem.